### PR TITLE
Introduce a query for obtaining database size

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -5701,6 +5701,15 @@ def cache_clear(self) -> None:
...
             method.cache_clear()
+            
+    def get_database_size(self) -> int:
+        """Get size of the database in MiB."""
+        query = f"SELECT pg_database_size(pg_database.datname) AS size_in_mb FROM "\
+                f"pg_database WHERE pg_database.datname='{self._engine.url.database}'"
+
+        with self._session_scope() as session:
+            result = session.execute(query).fetchone()
+            return result[0]
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:5712]: is it returning MB?

```
from thoth.storages import GraphDatabase
graph = GraphDatabase()
graph.connect()
graph.get_database_size()

17048657031
```

Comment (fridex) [reply thoth/storages/graph/postgres.py:5755]: In bytes, adjusted and rebased in the most recent commit. PTAL.

Comment (pacospace) [reply thoth/storages/graph/postgres.py:5755]: Thanks!!

**Review by pacospace — APPROVED** [review]
💯 🍻

**Pull request merged**